pkg/runner/cluster_swarm.go

```diff
@@ -117,8 +117,22 @@ func (*ClusterSwarmRunner) Run(ctx context.Context, input *api.RunInput, ow io.W
 		return nil, err
 	}
 
+	const maxNameLength = 63
+	var shortRunID string
+	prefix := fmt.Sprintf("tg-%s-%s-", input.TestPlan.Name, testcase.Name)
+	suffix := "-default"
+	availableChars := maxNameLength - len(prefix) - len(suffix)
+	if availableChars < 4 {
+		return nil, fmt.Errorf("test plan name + test case name too long")
+	}
+	if len(input.RunID) > availableChars {
+		shortRunID = input.RunID[len(input.RunID)-availableChars:]
+	} else {
+		shortRunID = input.RunID
+	}
```

Comment:
It'd be nice if this is extracted into a function that returns a short name that adheres to Docker Swarm's requirements of 63 chars, but I guess for now that works as well, as long as we don't start using it anywhere else :)